### DISCUSSION TITLE
Recreate Project Environment Variables BUG

### DISCUSSION THREAD

**Discussion by Nash-x9** [vercel/community#3325] [Bug Report]
### Page to Investigate

https://vercel.com/x/x/settings/environment-variables

### Steps to Reproduce

My first project had some bugs, so I deleted the project in vercel.

After deleting it I recreated my GitHUB repository and specified the same environment variables, which led to the beginning of the nightmare.

I realized that my environment variable didn't work, then I was going to delete the environment variable and redeploy the project, however I wasn't able to delete the environment variable, the page prompted me that my environment variable was from 15 days ago (i.e., the first project), and deleting it prompted `The Environment Variable "zoV7ksxQV1dIwvWM" was not found.`

**Comment by Nash-x9** [comment]
I deleted env successfully by Vercel CLI, and project work with new env.Vercel Web should check this.

**Reply by amyegan** [reply]
I'm glad you found a workaround! Thanks for sharing this. I sent your feedback to the UI team for investigation 🙇
